client/app/scripts/superdesk-desks/desks.js

```diff
@@ -275,6 +275,21 @@
 
                 var userDesks, userDesksPromise;
 
+                var _fetchCombined = function(endpoint, page, items) {
```

Comment:
Currently, desks service fetches all desks/stages/members and compiles lookup tables to use app-wide, and supposedly this should happen only once.

Ideally it needs to change, and fetch data only when needed and cache for future use. But until then, this needed to be done to be able to display stages.